postgresql-store.js

```diff
@@ -0,0 +1,170 @@
+const Assert = require('assert')
+const Pg = require('pg')
+
+const Q = require('./lib/qbuilder')
+const { intern } = require('./lib/intern')
+const { asyncMethod } = intern
+
+const STORE_NAME = 'postgresql-store'
+const ACTION_ROLE = 'sql'
+
+
+function postgres_store(options) {
+  const seneca = this
+
+  const {
+    fromColumnName = intern.identity,
+    toColumnName = intern.identity
+  } = options
+
+
+  let dbPool
+
+  function configure(spec, done) {
+    const conf = intern.getConfig(spec)
+
+    dbPool = new Pg.Pool({
+      user: conf.user,
+      host: conf.host,
+      database: conf.database,
+      password: conf.password,
+      port: conf.port
+    })
+
+    return done()
+  }
+
+  const store = {
+    name: STORE_NAME,
+
+    close: function (_msg, done) {
+      dbPool.end().then(done).catch(done)
+    },
+
+    save: asyncMethod(async function (msg) {
```

Comment:
is seneca-promisify required?

Reply:
No, `seneca-promisify` should not be required.